### PR TITLE
[IMP] l10n_pe: eCommerce electronic invoicing

### PR DESCRIPTION
**Pull request by gmz-odoo**
- Add new section for **eCommerce Functionality starting Odoo17+**
- Section will be display after the “_Guía de entrega electrónica 2.0”_ menu with the similar hierarchy
- **Pictures** on shared folder: https://drive.google.com/drive/folders/1fDgPxJFOpesZrkE3TsaOytDIOT0OW9Cf?usp=sharing

### CODE REVIEW

**Review by samueljlieber — APPROVED** [review]
Hi @gmz-odoo! Thanks for your work on this PR! I went ahead and added some of the images you provides as well as made some technical revisions on the RST added in this PR.

I left some images out that I felt where unnecessary, to help cut down on the overall number of images in this document, as well we are trying to not use markup in images wherever possible :)

Please review and let me know if you have any additional changes! Thank you!
Built documentation for this PR: https://runbot163.odoo.com/runbot/static/build/66422323-17-0/logs/build/html/applications/finance/fiscal_localizations/peru.html#ecommerce-electronic-invoicing

**Comment by gmz-odoo** [comment]
@samueljlieber 
AMAZING, I just had a look, and I think we are ready to go!
image

Thanks so much for your help =]

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 1]
```diff
@@ -870,3 +872,156 @@ Common errors
...
+button.
+
+If you want to manage the clients' access in the backend, review the
+:doc:`../../general/users/portal` documentation.
+
+Input fiscal fields in the checkout process
+*******************************************
+
+As mentioned above, fiscal field inputs are available in the checkout process. Entering fiscal data
+enables the purchase to conclude in the corresponding electronic document.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:1013]:
```suggestion
enables the purchase to create the corresponding electronic document.
```

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 2]
```diff
@@ -870,3 +872,156 @@ Common errors
...
+When the client makes a successful purchase and payment, the invoice is generated with the
+corresponding |EDI| elements, and the document type (Boleta/Factura) is selected based on the tax ID
+defined by the contact (RUC/DNI).
+
+Even though the invoice is created, it needs to :ref:`be sent to the OSE and the SUNAT
+<peru/electronic-invoice-status>`; that can be manually done per invoice, or the default scheduled
+action sends all published invoices once a day.
+
+Once the invoice is validated with |SUNAT|, the user can download the :file:`.zip` with the CDR, XML
+and PDF directly from the portal view.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:1024]:
```suggestion
Once the invoice is validated with |SUNAT|, the user can download the :file:`.zip` with the CDR,
XML, and PDF directly from the portal view by clicking the :guilabel:`Download` button.
```

Comment (dade-odoo) [reply content/applications/finance/fiscal_localizations/peru.rst:1024]: We're in the process of publishing new documentation guidelines, but one of the issues that we're implementing is to remove any markups from images. Also, we want the written documentation to be clear enough to stand on its own without screenshots, which it wasn't originally, but with the minor addition telling the user where to download the file, it should be clear enough to remove the screenshot.

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 3]
```diff
@@ -870,3 +872,156 @@ Common errors
...
+
+.. image:: peru/l10npe-edi-files-pe-ecommerce.png
+   :alt: Downloading electronic invoicing files from portal view.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:1027]:
```suggestion
```

**Comment by gmz-odoo** [comment]
Hello team,

**_Thanks_** @dade-odoo for the improvements with Grammar and optimizing the structure as expected.

- In recap, if you consider that eliminating some screenshots is needed, I understand it.

- The idea of having them explicitly on the User Doc is because we just run into a lot of issues with Users not being as familiar with the Odoo interface  sometimes, so we wanted to make it extra clear to avoid time spent on tickets/calls for very basic use of fields, but it was with the aim of minimizing frequent questions/errors due clarity.

- Suggestions, changes and improvements made by @dade-odoo , look good to me, and I'll also consider it for some next sections to be added for this and other LATAM localizations.

Let me know if you happen to need anything else from my side, I appreciate your help with this task.

Cheers, happy MiniFriyayy  _(from the SF side)_
@samueljlieber

**Comment by samueljlieber** [comment]
Thank you so much for your thorough review @dade-odoo! I implemented each of your suggestions in 8c5cd94. Thank you!

**Review by dade-odoo — APPROVED** [review]
Hey @gmz-odoo and @samueljlieber sorry for the delay! I was OOO. Looks great, passing on to the final review :)

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hello @gmz-odoo @samueljlieber thank you for your work on this! @gmz-odoo, I noticed that most configurations are already covered in the eCommerce documentation. To avoid duplicate content, we usually reference existing documentation rather than repeat it. Localization docs should focus on what's unique to that localization (or what isn't mentioned elsewhere in the documentation) since they're already quite extensive. I've added some suggestions in my review to adjust the PR accordingly. Thank you!

**Comment by auva-odoo** [comment]
@robodoo r+

**Comment by auva-odoo** [comment]
@robodoo retry

**Pull request closed without merging**